lib/WebpackConfig.js

```diff
@@ -326,14 +327,20 @@ class WebpackConfig {
 
         this.babelConfigurationCallback = callback;
 
-        for (const optionKey of Object.keys(options)) {
+        for (let optionKey of Object.keys(options)) {
```

Comment:
this could stay a const. You never re-assign it.

Reply:
Yep, I was re-assigning it in a previous version, forgot to change it back.